packages/cli/src/config/settingsSchema.ts

```diff
@@ -544,6 +544,16 @@ export const SETTINGS_SCHEMA = {
       'Enable AI-powered prompt completion suggestions while typing.',
     showInDialog: true,
   },
+  truncateToolOutput: {
+    type: 'boolean',
+    label: 'Truncate Tool Output',
+    category: 'General',
+    requiresRestart: false,
+    default: false,
```

Comment:
Off by default for now for evals? Assuming so what's the plan for swapping to on-by-default?

Reply:
The plan is to first run a small experiment for internal and external users to see if any major issues surface. Last time we tried rolling this out we had immediate pushback because the model did not get the necessary context.